subprojects/gradle/slack/build.gradle.kts

```diff
@@ -3,6 +3,7 @@ plugins {
     id("java-test-fixtures")
     `maven-publish`
     id("com.jfrog.bintray")
+    id("nebula.integtest")
```

Comment:
Can you please point to documentation or examples?
I haven't got it from https://github.com/nebula-plugins/nebula-project-plugin#nebula-integtest-plugin

Reply:
it is only in provided links (nebula plugin and official docs with concept)